"Parent" Unknown in tablegen properties of lldb Plugin

It seems as if `Parent` should be `Path` in lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDPProperties.td.

On macOS I cannot compile after cdbe28887bc9b4740f990dd5b3d8a86e9b14c131. 
```
lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDPProperties.td:3:32: error: Value 'Parent' unknown!
let Definition = "processkdp", Parent = "plugin.process.kdp" in {
                               ^
[4174/5751] Building CXX object tools/...nPlatformLinux.dir/PlatformLinux.cpp.o
ninja: build stopped: subcommand failed.
```

Changing `Parent` to `Path` makes everything compile correctly.

Thank you for everything!

--- a/lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDPProperties.td
+++ b/lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDPProperties.td
@@ -1,6 +1,6 @@
 include "../../../../include/lldb/Core/PropertiesBase.td"
 
-let Definition = "processkdp", Parent = "plugin.process.kdp" in {
+let Definition = "processkdp", Path = "plugin.process.kdp" in {
   def KDPPacketTimeout: Property<"packet-timeout", "UInt64">,
     Global,
     DefaultUnsignedValue<5>,

--- a/lldb/source/Plugins/Trace/intel-pt/TraceIntelPTProperties.td
+++ b/lldb/source/Plugins/Trace/intel-pt/TraceIntelPTProperties.td
@@ -1,6 +1,6 @@
 include "../../../../include/lldb/Core/PropertiesBase.td"
 
-let Definition = "traceintelpt", Parent = "plugin.trace.intel-pt" in {
+let Definition = "traceintelpt", Path = "plugin.trace.intel-pt" in {
   def InfiniteDecodingLoopVerificationThreshold:
       Property<"infinite-decoding-loop-verification-threshold", "UInt64">,
     Global,

